### ISSUE TITLE
Redeploy on WildFly with the GraphQL Feature Pack should reload the config

### ISSUE THREAD

**Issue by t1** [smallrye/smallrye-graphql#1591]
When we redeploy a `war` on a WildFly with an update to the `microprofile-config.properties` concerning MP GraphQL, e.g. `mp.graphql.showErrorMessage`, then the changes don't get applied; it needs a full restart for that.

I've created a reproducer on https://github.com/t1/graphql-config-reload-reproducer including instructions for reproducing.

**Comment by jmartisk** [comment]
Yeah but redeploying also counts as multiple deployments in this regard.
If it works correctly with other MP Config settings though, it's probably something that we can fix in SmallRye GraphQL.

**Comment by t1** [comment]
I've updated the reproducer to show how updates to the config for MP REST Client works after a redeploy, but that may not be comparable, as it's a client, not a server.
So I've also tried to add tags to MP Metrics via MP Config `mp.metrics.tags`, but it doesn't work... I probably do something wrong.
